wordpress-mobile/WordPress-Android
# Jetpack App (Basics): Update App Icons

Fixes #14305

This PR updates the `Jetpack` app icons based on the latest app icons provided by design.

App | Launcher Icons
-----|-----
WordPress | <img width="750" height="300" alt="wordpress_launcher_icons" src="https://user-images.githubusercontent.com/9729923/114701236-3697ba80-9d2b-11eb-8369-93dbe8f598f6.png">
Jetpack | <img width="750" height="300" alt="jetpack_launcher_icons" src="https://user-images.githubusercontent.com/9729923/114701296-4b744e00-9d2b-11eb-9004-5e7bae6e7199.png">

**To Test**
- Build and install all the `Jetpack` variants (`prod`, `beta` and `alpha`):
  - `jetpackVanillaDebug` for `prod`,
  - `jetpackWasabiDebug` for `beta`, and
  - `jetpackJalapenoDebug` for `alpha`.
- Verify that the app icons are updated and match the expected design.

**Merge Instructions**
- ~~A quick design review by @osullivanchris on the app icons changes for the Jetpack app.~~ 🟢 
- ~~Remove `Needs Design Review` label.~~ 🟢 
- Merge as normal.

## Regression Notes
1. Potential unintended areas of impact 🟢 
2. What I did to test those areas of impact (or what existing automated tests I relied on) 🟢 
3. What automated tests I added (or what prevented me from doing so) 🟢 

PR submission checklist:
